src/components/Faq/index.tsx

```diff
@@ -0,0 +1,29 @@
+export interface FaqProps {
+    question: string;
+    answer: string;
+}
+
+const Sponsor: React.FC<FaqProps> = ({ question, answer }) => {
+    return (
+        <div className="flex flex-col justify-center">
+            <div className="my-4">
+                <p
+                    className="font-bold text-3xl text-orange-600 text-center "
```

Comment:
Maybe its a better ideia to justify the text on PC to the left and on Mobile keep it on the center. 

_Try to replicate the Figma design and pay attention to the font sizes and font text_